src/components/Homepage/HomePage.test.js

```diff
@@ -0,0 +1,83 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+import axios from "axios";
+import { MemoryRouter } from "react-router-dom";
+import HomePage from "./HomePage";
+import "@testing-library/jest-dom";
+
+jest.mock("axios");
+
+const MockHomePage = () => {
+  return (
+    <MemoryRouter>
+      <HomePage />
+    </MemoryRouter>
+  );
+};
+
+describe("Homepage", () => {
+  let response;
+  beforeEach(() => {
+    response = {
+      data: {
+        results: [
+          { title: "first", id: 1 },
```

Comment:
Can we use data, which is similar to what we receive from the API response?

Reply:
fixed